examples/leaflet_local/filtered.html

```diff
@@ -0,0 +1,140 @@
+<html>
+<head>
+    <link rel="stylesheet" href="https://unpkg.com/leaflet@1.6.0/dist/leaflet.css" />
+    <link rel="stylesheet" href="/examples/site.css" />
+    <script src="https://unpkg.com/underscore@1.13.1/underscore-min.js"></script>
+    <script src="https://unpkg.com/leaflet@1.6.0/dist/leaflet.js"></script>
+    <script src="/dist/flatgeobuf-geojson.min.js"></script>
+    <script src="https://unpkg.com/json-formatter-js"></script>
+
+    <style>
+        #map { height: 480px; }
+    </style>
+</head>
+<body>
+    <ul class="primary-navigation">
+        <li class="active">
+            Leaflet Example
+        </li>
+        <li>
+            <a href="/examples/openlayers_local/">OpenLayers Example</a>
+        </li>
+    </ul>
+    <ul class="secondary-navigation">
+        <li><a href="/examples/leaflet_local/">Basic Example</a></li>
+        <li class="active">Filter By Rect</li>
+        <li><a href="/examples/leaflet_local/large.html">Filtering a Large Dataset</a></li>
+    </ul>
+
+    <div id="map"></div>
+    <script>
+        document.addEventListener("DOMContentLoaded", async () => { 
```

Comment:
For purposes of example's, I think it's just a little easier for people to see the script in-lined into the HTML, so everything they need to get started can be copy/pasted from one spot.

Is that alright?

Reply:
Agreed I will change the existing ones to that.